README.md

```diff
@@ -1,3 +1,31 @@
-# Memes
+# memes.skyra.pw
 
 This is a private dynamic meme generator designed primarily for updating [Artiel](https://github.com/skyra-project/artiel)'s meme template file.
+
+## Setup
+
+To run the database locally, you need to use and configure Wrangler:
+
+```toml
+# wrangler.toml
+name = "memes-skyra-pw"
+main = "./.output/server/index.mjs"
+compatibility_date = "2023-05-18"
+
+[[d1_databases]]
+binding = "DATABASE"
```

Comment:
this is not a template string, is it?

Reply:
That is not, no, binding is what property from `process.env` wrangler will inject the `D1Database` instance.